### ISSUE TITLE
Run survey jobs in more of trickle rather than a deluge

### ISSUE THREAD

**Issue by kurtwheeler** [AlexsLemonade/refinebio#1480]
### Context

When we run 200 surveyor jobs all at the same time we kill the database:

![image](https://user-images.githubusercontent.com/4026833/62894516-d3b27c80-bd1a-11e9-8411-393969d9c9ef.png)

### Problem or idea

We should run surveyor jobs 1-5 at a time. This will probably generate data faster than we can process it, but without knocking over our own database so hard.

### Solution or next step

I'm not 100% sure on how we should implement this just yet. I know it needs to happen though. The simplest naive solution I can think of would be to de-parallelize survey jobs, let each one do a list of experiment accessions one at a time, and then we can just choose how many to run simultaneously.

**Comment by arielsvn** [comment]
Looking at the logs, there're a lot of queries like these:

```
2019-08-12 20:00:04 UTC:10.0.73.172(37070):drpostgresuser@data_refinery:[21349]:LOG:  duration: 1866.064 ms  statement: SELECT "original_files"."id", "original_files"."filename", "original_files"."absolute_file_path", "original_files"."size_in_bytes", "original_files"."sha1", "original_files"."s3_bucket", "original_files"."s3_key", "original_files"."source_url", "original_files"."is_archive", "original_files"."source_filename", "original_files"."has_raw", "original_files"."is_downloaded", "original_files"."is_public", "original_files"."created_at", "original_files"."last_modified" FROM "original_files" WHERE ("original_files"."has_raw" = true AND "original_files"."source_filename" = 'SRR6652790' AND "original_files"."source_url" = 'dbtest@sra-download.ncbi.nlm.nih.gov:data/sracloud/traces/sra58/SRR/006496/SRR6652790')
2019-08-12 20:00:04 UTC:10.0.73.172(37064):drpostgresuser@data_refinery:[21304]:LOG:  duration: 1868.728 ms  statement: SELECT "original_files"."id", "original_files"."filename", "original_files"."absolute_file_path", "original_files"."size_in_bytes", "original_files"."sha1", "original_files"."s3_bucket", "original_files"."s3_key", "original_files"."source_url", "original_files"."is_archive", "original_files"."source_filename", "original_files"."has_raw", "original_files"."is_downloaded", "original_files"."is_public", "original_files"."created_at", "original_files"."last_modified" FROM "original_files" WHERE ("original_files"."has_raw" = true AND "original_files"."source_filename" = 'SRR7475924.sra' AND "original_files"."source_url" = 'anonftp@ftp-private.ncbi.nlm.nih.gov:/sra/sra-instant/reads/ByRun/sra/SRR/SRR747/SRR7475924/SRR7475924.sra')
2019-08-12 20:00:04 UTC:10.0.73.172(36980):drpostgresuser@data_refinery:[4623]:LOG:  duration: 1728.542 ms  statement: SELECT "original_files"."id", "original_files"."filename", "original_files"."absolute_file_path", "original_files"."size_in_bytes", "original_files"."sha1", "original_files"."s3_bucket", "original_files"."s3_key", "original_files"."source_url", "original_files"."is_archive", "original_files"."source_filename", "original_files"."has_raw", "original_files"."is_downloaded", "original_files"."is_public", "original_files"."created_at", "original_files"."last_modified" FROM "original_files" WHERE ("original_files"."has_raw" = true AND "original_files"."source_filename" = 'SRR6741494.sra' AND "original_files"."source_url" = 'anonftp@ftp.ncbi.nlm.nih.gov:/sra/sra-instant/reads/ByRun/sra/SRR/SRR674/SRR6741494/SRR6741494.sra')
2019-08-12 20:00:04 UTC:10.0.73.172(37056):drpostgresuser@data_refinery:[21250]:LOG:  duration: 1829.790 ms  statement: SELECT "original_files"."id", "original_files"."filename", "original_files"."absolute_file_path", "original_files"."size_in_bytes", "original_files"."sha1", "original_files"."s3_bucket", "original_files"."s3_key", "original_files"."source_url", "original_files"."is_archive", "original_files"."source_filename", "original_files"."has_raw", "original_files"."is_downloaded", "original_files"."is_public", "original_files"."created_at", "original_files"."last_modified" FROM "original_files" WHERE ("original_files"."has_raw" = true AND "original_files"."source_filename" = 'SRR6798782' AND "original_files"."source_url" = 'dbtest@sra-download.ncbi.nlm.nih.gov:data/sracloud/traces/sra60/SRR/006639/SRR6798782')
2019-08-12 20:00:05 UTC:10.0.73.172(37068):drpostgresuser@data_refinery:[21347]:LOG:  duration: 1719.148 ms  statement: SELECT "original_files"."id", "original_files"."filename", "original_files"."absolute_file_path", "original_files"."size_in_bytes", "original_files"."sha1", "original_files"."s3_bucket", "original_files"."s3_key", "original_files"."source_url", "original_files"."is_archive", "original_files"."source_filename", "original_files"."has_raw", "original_files"."is_downloaded", "original_files"."is_public", "original_files"."created_at", "original_files"."last_modified" FROM "original_files" WHERE ("original_files"."has_raw" = true AND "original_files"."source_filename" = 'SRR6983547' AND "original_files"."source_url" = 'dbtest@sra-download.ncbi.nlm.nih.gov:data/sracloud/traces/sra61/SRR/006819/SRR6983547')
```

https://github.com/AlexsLemonade/refinebio/pull/1472 Might help with these since it adds an index on `source_filename` for original files

**Comment by cgreene** [comment]
Closed with #1500
